pkg/apis/eksctl.io/v1alpha5/cloudwatch.go

```diff
@@ -17,16 +17,23 @@ const (
 	wildcardLogging          = "*"
 )
 
+var LogRetentionInDaysValues = []int{1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653}
+
 // ClusterCloudWatchLogging container config parameters related to cluster logging
 type ClusterCloudWatchLogging struct {
 
 	// Types of logging to enable (see [CloudWatch docs](/usage/cloudwatch-cluster-logging/#clusterconfig-examples)).
 	// Valid entries are `CloudWatchLogging` constants
 	//+optional
 	EnableTypes []string `json:"enableTypes,omitempty"`
+	// LogRetentionInDays sets the number of days to retain the logs for (see [CloudWatch docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html)) .
+	// Valid values are: 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731,
+	// 1827, and 3653.
+	//+optional
+	LogRetentionInDays int `json:"logRetentionInDays,omitempty"`
```

Comment:
I couldn't see anything in the doc, are those really the only valid values? Feels so random :joy: